content/sensu-go/5.11/sensuctl/reference.md

```diff
@@ -329,6 +330,23 @@ To create the `pagerduty` handler in the current session namespace:
 sensuctl create --file pagerduty.yml
 {{< /highlight >}}
 
+## Deleting resources
+
+The `sensuctl delete` command allows you to delete resources by reading from STDIN or a flag configured file (`-f`).
+The `delete` command accepts Sensu resource definitions in `wrapped-json` and `yaml` and uses the same [resources types][3] as `sensuctl create`.
+
+To delete all resources from `my-resources.yml` using `sensuctl delete`:
+
+{{< highlight shell >}}
+sensuctl delete --file my-resources.yml
+{{< /highlight >}}
```

Comment:
Does `sensuctl delete` support multi-namespace functionality similar to [`sensuctl create`](https://docs.sensu.io/sensu-go/5.10/sensuctl/reference/#creating-resources-across-namespaces)?

Reply:
Good question. I think so, I need to look into this before confirming.

Reply:
Confirmed!